src/lib_proj.cmake

```diff
@@ -438,6 +438,13 @@ set_target_properties(proj
 set(PROJ_LIBRARIES proj)
 # hack, required for test/unit
 set(PROJ_LIBRARIES ${PROJ_LIBRARIES} PARENT_SCOPE)
+if(WIN32)
+target_link_libraries (proj
+  PRIVATE
+    shell32.lib
+    ole32.lib
```

Comment:
This change shouldn't be needed: Both libs are members of `CMAKE_C_STANDARD_LIBRARIES_INIT` for Window. CMake will link them anyways.  
https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_STANDARD_LIBRARIES.html  
https://github.com/Kitware/CMake/blob/316bb67ff74e89a87e50370d809429f122689e3d/Modules/Platform/Windows-MSVC.cmake#L256